Aptoide/aptoide-client-v8
# [ASV-1807] load more home

**What does this PR do?**

   This PR aims at implementing the new error for home load more.

**Database changed?**

 No

**Where should the reviewer start?**


**How should this be manually tested?**

Open Home, turn off wifi, go to load more section. See the error. Turn internet on again, press retry and it should load the next items from home.

**What are the relevant tickets?**

  Tickets related to this pull-request: [ASV-1807](https://aptoide.atlassian.net/browse/ASV-1807)

**Questions:**

   Does this add new dependencies which need to be added? (Eg. new keys, etc.) 



**Code Review Checklist**
